web/css/styles.css

```diff
@@ -0,0 +1,75 @@
+.social:hover {
```

Comment:
Normally this kind of files are rendered by an optimizer, so the source files (maybe written in scss) are somewhere else and a script (maybe gulp or assetic-bundle) optimizes the sources to a single file and puts it here.
I advice to do it this way (even if it's does copy for now)